harvester/HarvestRepository.py

```diff
@@ -22,7 +24,7 @@ def __init__(self, params):
 	def setDefaults(self, defaults):
 		""" Inherit global configs, but they do not overwrite local configs """
 		for k,v in defaults.items():
-			if isinstance(k, str) or isinstance(k, unicode):
+			if isinstance(k, str) or isinstance(k, unicode) and not isinstance(v, list):
```

Comment:
The global configuration lists being put into the repo configs were messing up my debug logs.  But in hindsight, I can simplify that, hang on.